assemblyline_v4_service/common/task.py

```diff
@@ -7,7 +7,7 @@
 from assemblyline.common import forge
 from assemblyline.common import log as al_log
 from assemblyline.common.classification import Classification
-from assemblyline.common.digests import get_sha256_for_file
+from assemblyline.common.digests import get_digests_for_file, get_sha256_for_file
```

Comment:
You probably don't need **get_sha256_for_file** import anymore

Reply:
Will keep it for functions that only rely on the sha256 for validation, namely `download_file` but also `_add_file`